src/views/index/index.jsx

```diff
@@ -81,19 +81,19 @@ const Index = () => {
                 {/* end content-graphic */}
                 <div className="content-news">
                     <div className="content-news-header">
-                        News
+                       News
                     </div>
                     <div className="content-news-body">
                         <a
-                            href="https://medium.com/scratchfoundation-blog/10-things-to-try-right-now-with-your-child-on-scratch-and-scratchjr-defd7f8ad0d1"
+                            href="https://www.amazon.com/Official-ScratchJr-Book-Help-Learn/dp/1593276710/"
                         >
-                        10 things to try right now with your child on Scratch and ScratchJr
+                      The Official ScratchJr Book is the perfect companion to the app and makes coding easy and fun for all
                         </a>, hands-on activities for getting started with ScratchJr.
                     </div>
                     <div className="content-news-body">
-                        Check out tips for creating with your child: <a href="https://medium.com/scratchfoundation-blog/creating-together-3041662c970a">
-                        Creating Together
-                        </a> by the Scratch Foundation.
+                       <a href="https://www.amazon.com/ScratchJr-Coding-Cards-Creative-Activities/dp/1593278993/"> The ScratchJr Coding Cards are a deck of 75 activity cards covering fun and exciting projects designed to educate young children with ScratchJr.
+                        
```

Comment:
Close the `</a>` link after 'Cards', and have the rest of the text follow the link.